feat(changelog): apple-silicon-added-introduction-of-a-new-monthly-confi 2025-04-07 (#4804)
scaleway/docs-content: changelog/april2025/2025-04-07-apple-silicon-added-introduction-of-a-new-monthly-confi.mdx

---
title: Introduction of a new monthly configuration offering discounted prices
status: added
date: 2025-04-07
category: bare-metal
product: apple-silicon
---

> Monthly commitment plan for Apple silicon is a new billing plan to enjoy better rates than pay-as-you-go pricing.

Reviewer: ```suggestion
The monthly commitment plan for Apple silicon is a new billing plan, offering better rates than pay-as-you-go pricing.
```

The initial commitment is for one month from the server order. At the end of your initial commitment period, the commitment is automatically renewed every month on the anniversary date unless you choose to delete the machine, in which case the deletion will take effect on the anniversary date of the commitment.

Monthly pricing:
- M1-M: €79/month 
- M2-M: €115/month
- M2-L: €159/month
- M4-S: €149/month

> Read [our documentation](https://www.scaleway.com/en/docs/apple-silicon/how-to/manage-commitment-plan/) on how to manage Commitment plans.

Reviewer: ```suggestion
Read [our documentation](https://www.scaleway.com/en/docs/apple-silicon/how-to/manage-commitment-plan/) on how to manage commitment plans.
```
